homeassistant/components/screenlogic/switch.py

```diff
@@ -31,7 +36,8 @@ async def async_setup_entry(
                 coordinator,
                 circuit_num,
                 CODE.STATUS_CHANGED,
-                circuit["name"] not in GENERIC_CIRCUIT_NAMES,
+                circuit["name"] not in GENERIC_CIRCUIT_NAMES
+                and circuit["interface"] != INTERFACE_GROUP.DONT_SHOW,
```

Comment:
I think it would make sense to change them to entity_registry_enabled_default=False instead in case they do want to use them, and we avoid a breaking change

Reply:
That was the intention. Is it not?
https://github.com/home-assistant/core/blob/4f4defc0558fa3ca4c4b364e76bc84baa2bc7856/homeassistant/components/screenlogic/entity.py#L26

Reply:
That is what its doing. I read it incorrectly on my mobile.  Sorry about that.

Reply:
No worries. Would it help to rename that parameter to something like `default_enable` ?

Reply:
Switching to SensorEntityDescription would help as it uses named kwargs.